tests/Doctrine/Tests/Common/Collections/BaseArrayCollectionTest.php

```diff
@@ -269,6 +269,20 @@ public function testExists() : void
         }), 'Element not exists');
     }
 
+    public function testFind() : void
```

Comment:
Please split up into separate tests for "found" and "not found".

Reply:
All the others tests are not split.
- `testRemove` try to remove 5 elements
- `testEmpty` is called on empty and not empty
- `testExist` is almost a copy paste of my test

And so on...

I feel like it would be inconsistent to split this one.

Reply:
Should have been, but fixing that is out of scope for your PR.